modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc

```diff
@@ -161,126 +161,9 @@ rosa create cluster --sts
-+
---
-** If you deployed your cluster using a single availability zone, set the *Minimum node count* and *Maximum node count*. This defines the minimum and maximum compute node limits in the availability zone.
-** If you deployed your cluster using multiple availability zones, set the *Minimum nodes per zone* and *Maximum nodes per zone*. This defines the minimum and maximum compute node limits per zone.
---
-+
-[NOTE]
-====
-Alternatively, you can set your autoscaling preferences for the default machine pool after the machine pool is created.
-====
-
-. If you did not enable autoscaling, select a compute node count for your default machine pool:
-** If you deployed your cluster using a single availability zone, select a *Compute node count* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool for the zone.
-** If you deployed your cluster using multiple availability zones, select a *Compute node count (per zone)* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool per zone.
-
-. Optional: Expand *Edit node labels* to add labels to your nodes. Click *Add label* to add more node labels and select *Next*.
-
-. In the *Cluster privacy* section of the *Network configuration* page, select *Public* or *Private* to use either public or private API endpoints and application routes for your cluster.
-+
-[IMPORTANT]
-====
-If you are using private API endpoints, you cannot access your cluster until you update the network settings in your cloud provider account.
-====
-
-. Optional: If you opted to use public API endpoints, you can select *Install into an existing VPC* to install your cluster into an existing VPC.
-+
-[NOTE]
-====
-If you opted to use private API endpoints, you must use an existing VPC and PrivateLink and the *Install into an existing VPC* and *Use a PrivateLink* options are automatically selected. With these options, the Red Hat Site Reliability Engineering (SRE) team can connect to the cluster to assist with support by using only AWS PrivateLink endpoints.
-====
-
-. Optional: If you are installing your cluster into an existing VPC, select *Configure a cluster-wide proxy* to enable an HTTP or HTTPS proxy to deny direct access to the internet from your cluster.
-
-. Click *Next*.
-
-. If you opted to install the cluster in an existing AWS VPC, provide your *Virtual Private Cloud (VPC) subnet settings*.
-+
-[NOTE]
-====
-You must ensure that your VPC is configured with a public and a private subnet for each availability zone that you want the cluster installed into. If you opted to use PrivateLink, only private subnets are required.
-====
+. On the *Cluster details* page, provide a *Cluster name*. Select the default values for the remaining fields and click *Next*.
```

Comment:
Outside the scope, but in Step 2, you may want to specify that it's under the "Managed services" section, though that may just be the way my account is provisioned

![image](https://user-images.githubusercontent.com/16167833/185397499-6caefa69-8c4a-406a-bb36-3ee13fbb9974.png)

Reply:
On Step 4, if your account _is_ associated and roles were created, you have a series of role ARNs for these roles presented. I don't think you need to mention it, but it did catch me off guard a bit. I see you refer to this in Step 5. I'm unsure if that flow works since if I am going through this with an account that is already configured, I'd likely get lost by Step 4, but again, outside the scope of your changes. Maybe a "to think about for later" item.

![image](https://user-images.githubusercontent.com/16167833/185398454-a7656011-7299-4b54-8658-398f455dff71.png)

Reply:
> Outside the scope, but in Step 2, you may want to specify that it's under the "Managed services" section, though that may just be the way my account is provisioned

@EricPonvelle thanks for your review and suggestions. For this one, I didn't mention the section on the page because I thought it was surplus to requirements. There is only one ROSA row from which you can create a cluster and there are not many sections on the page. Do you think it is necessary to add it? I was opting in favour of providing only the required details to minimise the wording in the step.

Reply:
> On Step 4, if your account _is_ associated and roles were created, you have a series of role ARNs for these roles presented. I don't think you need to mention it, but it did catch me off guard a bit. I see you refer to this in Step 5. I'm unsure if that flow works since if I am going through this with an account that is already configured, I'd likely get lost by Step 4, but again, outside the scope of your changes. Maybe a "to think about for later" item.

@EricPonvelle I have updated the procedure to try to address this item. Please let me know if this resolves the issue. I will then run further testing and update the customisation section for consistency.

Reply:
After further testing in an environment that starts with no account roles, I have updated both the quick and customised procedures through this PR to address the items in this thread.